benchmark/domain/domain-fn-args.js

```diff
@@ -12,14 +12,14 @@ var gargs = [1, 2, 3];
 
 function main(conf) {
 
-  var args, n = +conf.n;
```

Comment:
Nit: any reason for putting this in for block instead of keeping it here on a separate line?